### DISCUSSION TITLE
按照文档在windows环境下部署完只能使用截图功能

### DISCUSSION THREAD

**Discussion by Sakuararua** [hiroi-sora/Umi-OCR#716] [Q&A]
截图功能可以正常进行OCR识别，但是pdf识别和图片的导入识别程序会直接崩溃退出

**Comment by hiroi-sora** [comment]
没有什么错误提示或弹窗吗？

通过 `UmiOCR-data\RUN_CLI.bat` 启动，看看控制台里有没有报错？
